.github/workflows/update.yaml

```diff
@@ -78,6 +78,11 @@ jobs:
           # Convert main strings to XLIFF
           python translationFiles/.github/scripts/extract_source_strings.py --input translations.ts --output translationFiles/en/mozillavpn.xliff
 
+          # Copy the extras.xliff file over
+          if [ -d "vpn/src/translations/extras/extras.xliff" ]; then
```

Comment:
Note to self: this wasn't doing anything, since it's check for a folder, not a file.